x/evidence/keeper/grpc_query.go

```diff
@@ -22,10 +22,6 @@ var _ types.QueryServer = Keeper{}
 
 // Evidence implements the Query/Evidence gRPC method
 func (k Keeper) Evidence(c context.Context, req *types.QueryEvidenceRequest) (*types.QueryEvidenceResponse, error) {
-	if req == nil {
```

Comment:
Why were these removed?

Reply:
these lines are never used even when we pass empty request weirdly, it just passes to the next check

Reply:
If we re-add them, we should re-add it in NFT too: https://github.com/cosmos/cosmos-sdk/pull/14055/files#diff-b886f35de8a2d704d1dfdb9e7ee1077982831f7dc12733f803330bee76ae4d34L17

Reply:
okay, I can add that